OpenChart/src/Charting/BPM.cs

```diff
@@ -5,7 +5,7 @@ namespace OpenChart.Charting
     /// <summary>
     /// Represents a BPM (beats per minute) change in a chart.
     /// </summary>
-    public class BPM : Timing
+    public class BPM
```

Comment:
Is there any value (heh) in adding convenience operator overloads for these classes?

Reply:
The nice thing about the operator overloads is for stuff like a constructor that takes a Beat but you pass it a plain double. That doesn't really work for BPMs since a BPM object is more than one value